File: 은빈/자바/[PGS]피자나눠먹기2.java
class Solution {
    public int solution(int n) {
        int gcd = getGcd(Math.max(n, 6), Math.min(n, 6));
        int answer = n * 6 / gcd;

        answer /= 6;

        return answer;
    }

    private int getGcd(int num1, int num2) {
        int r = num1 % num2;

        if(r == 0) {
            return num2;
        }
        return getGcd(num2, r);
    }
}

File: 은빈/파이썬/[PGS]피자나눠먹기2.py
def getGcd(num1, num2):
    r = num1 % num2
    if(r == 0):
        return num2
    return getGcd(num2, r)

def solution(n):
    gcd = getGcd(max(n, 6), min(n, 6))
    answer = n * 6 / gcd

    answer /= 6

    return answer